docs/competition/matches/match-scoring.md

```diff
@@ -101,4 +101,10 @@ The steps to perform are as follows:
 
 There are two score sheets for every match. It's likely that there may be a discrepancy between the sheets. If that's the case, one match score entry volunteer should speak to the marshals to try to find the correct state.
 
-If you have any questions, speak to the head match scorer. In SR2019 the head match scorer is on the balcony.
+If you have any questions, speak to the head match scorer. In SR2023 the head match scorer can be called down from the balcony.
```

Comment:
Who is the head match scorer? In SR2022, we didn't have a head match scorer due to the change in how match ops worked. Is this more referring to myself/Alistair now?

Reply:
Good point, have changed it to the Judge/Competition Software Co-ordinator, based  on the issue.